kitsune/questions/migrations/0006_ios_questionlocale.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def create_questionlocale(apps, schema_editor):
+    Product = apps.get_model('products', 'Product')
+    QuestionLocale = apps.get_model('questions', 'QuestionLocale')
+
+    p = Product.objects.get_or_create(slug='ios', defaults={
+        'title': 'Firefox for iOS',
+        'description': 'Firefox for iPhone, iPad and iPod touch devices',
+        'display_order': 0,
+        'visible': False})
+
+    QuestionLocale.objects.get_or_create(locale='en-US', product=p)
```

Comment:
Products is a many-to-many field. You'll have to check the return value of get_or_create (which is `obj, was_created`), and add `p` the new object if it was created.